VideoLocker/src/main/java/org/edx/mobile/model/api/RegisterResponse.java

```diff
@@ -12,6 +16,19 @@
     private @SerializedName("field") String field;
     private @SerializedName("value") String value;
 
+    /* Registration Field error messages */
+    private @SerializedName(IRegistrationFieldView.FieldName.PASSWORD) List<RegisterResponseFieldError> passwordErrors;
```

Comment:
This should just be a Map from a field name ``String`` to a ``List<RegisterResponseFieldError>``. We don't want to hard code the possible fields.